backend/src/email/utils/callback/parsers/ses.ts

```diff
@@ -12,6 +12,7 @@ import {
 import { addToBlacklist } from '@email/utils/callback/query'
 import config from '@core/config'
 import { compareSha256Hash } from '@shared/utils/crypto'
+import { EmailTransactionalService } from '@email/services/email-transactional.service'
```

Comment:
truth be told, don't really understand what's going on in this file... seems to me that it's used to parse the callbacks from SES?